#!/usr/bin/env python
PACKAGE = "mpc_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("land_home", bool_t, 0, "MAV should land to the home position", False)
gen.add("land", bool_t, 0, "MAV will land ASAP", False)
gen.add("take_off", bool_t, 0, "MAV will take off ASAP", False)
exit(gen.generate(PACKAGE, "mpc_ros", "Commands"))
